Index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>Microverse Trail || Tribute Page</title>
+  </head>
+  <body>
+    <main id="main">
+      <div id="heading" class="center">
+        <h1 id="title" class="center">Mahatma Gandhi</h1>
+        <p class="font">
+          <i>mahatma gandhi the man who became one with the universal being</i>
+        </p>
+		<!--
+			<p class="font">
+          <i>mahatma gandhi the man who became one with the universal being</i>
+        </p>
+		-->
```

Comment:
Please do not leave unused code as a comment in your codebase. Unused code as comments make your code not clean.